python/deepsulci/sulci_labeling/capsul/labeling.py

```diff
@@ -139,6 +152,13 @@ def _run_process(self):
 
         graph['label_property'] = 'label'
         # save graph
-        aims.write(graph, self.labeled_graph,
+        aims.write(graph, self.labelled_graph,
                    options={"save_only_modified": False})
+        # since some cuts have been made, we need to update the voronoi and attributes that depend on it.
+        if summary['cuts'] != 0:
```

Comment:
I see that I did not put `self.rebuild_attributes` in the `if` like above but maybe I should, honestly I don't really know why we would want not to rebuild the attributes.

Reply:
It was just a debug feature which allowed to run the process faster, but for users, yes, it should always be enabled. I'm OK to remove it.